pkg/apiserver/database/demo.go

```diff
@@ -568,6 +569,7 @@ func createScans(assets []models.Asset, scanConfigs []models.ScanConfig) []model
 		JobsCompleted:          utils.PointerTo[int](1),
 		JobsLeftToRun:          utils.PointerTo[int](1),
 		TotalExploits:          utils.PointerTo[int](2),
+		TotalInfoFinder:        utils.PointerTo[int](2),
```

Comment:
Please add demo data findings (similar to createRootkitFindings())

Reply:
done https://github.com/openclarity/vmclarity/pull/623/commits/7acda5a7f3b280096e15a8a302aa888352ac4dda